packages/blade-old/src/icons/HyperLink/HyperLink.native.js

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { Path } from 'react-native-svg';
+import Icon, { IconPropTypes } from '../../atoms/Icon';
+
+function HyperLink(props) {
+  return (
+    <Icon viewBox="0 0 20 20" {...props}>
```

Comment:
```suggestion
    <Icon viewBox="0 0 24 24" {...props}>
```
@shamshad17 this should be the viewbox like in other icons, don't edit it to 24 directly here. It should be imported this way from the icon. Can you point us to the icon where you are importing it from? // @shridhar52war

Reply:
[Figma](https://www.figma.com/file/4GTOYOoG28EwW3mFXJh1Ik/Mobile-app-Navigation?node-id=2206%3A110788)

If I change viewbox to 24 icon becomes smaller in comparison to other icons. @shubhnik

Reply:
@shamshad17 we don't explicitly edit this property, it is exported from Figma directly. // @shridhar52war

Reply:
yes, @shubhnik rightly mentioned. @shamshad17 While exporting the svg, expose it with view box of "24 24".

Reply:
Got it. I'll ask designer to update the svg.

Reply:
@shamshad17 btw we already have a link icon, looks like it is similar and should suffice? https://github.com/razorpay/blade/tree/master/packages/blade-old/src/icons/Link @gupta-ji6

Reply:
@shubhnik Currently we have this Link icon.
<img width="24" alt="Screenshot 2022-03-11 at 3 10 11 PM" src="https://user-images.githubusercontent.com/88472926/157841867-4d89e1c3-c410-4f41-8f54-4ea7aeebbcf1.png">